test-utils/docs/default-roles.md

```diff
@@ -0,0 +1,71 @@
+# Default roles
+
+There are list of default roles in `@testing-library/react`:
```

Comment:
Do we really need to keep this list ? You can just leave a link to it in the right place